### ISSUE TITLE
radio script

### ISSUE THREAD

**Issue by tikismoke** [zadmalck/scripts-bash#1]
Hi juste be carreful mixing datatype....
Your script seems to be a DT_trigger except with params (volume) that seems to return a DT_string.

**Comment by zadmalck** [comment]
hi
./action_webradio volume 

I invite you to execute the bash script with a parameter volume ... :-)

**Comment by tikismoke** [comment]
Yes that's what i saw in code. But this mean it's à mixted script. So just be careful with it's implementation. You should add this advise in top of file.

**Comment by zadmalck** [comment]
root@raspberrypi:~# test=$(/env/action_webradio volume)
root@raspberrypi:~# echo "[$test]"
[80]
